src/schema/imported/manifest.json

```diff
@@ -162,13 +162,17 @@
               "type": "array",
               "default": [],
               "items": {
-                "anyOf": [
+                "allOf": [
                   {
                     "$ref": "#/types/Permission"
                   },
                   {
-                    "type": "string",
-                    "deprecated": "Unknown permission ${value}",
+                    "onError": "warn"
+                  }
+                ],
+                "anyOf": [
+                  {},
+                  {
                     "format": "deprecated"
                   }
```

Comment:
this `anyOf` block looks weird? Or is both together essentially warning with a deprecation error or something?